src/components/MultipleLeftRightSelector/MultipleLeftRightSelector.tsx

```diff
@@ -0,0 +1,226 @@
+/* Disabled lint due to incompatibility of Button with regular props, work on this is in progress */
+/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+// @ts-nocheck
+
+import React, {useState, useCallback, useRef} from 'react';
+import {ValueList} from './ValueList';
+import {Button} from '~/components';
+import {ChevronDoubleLeft, ChevronDoubleRight} from '~/icons';
+import {MultipleLeftRightSelectorProps, Option} from './MultipleLeftRightSelector.types';
+import './MultipleLeftRightSlector.scss';
+
+const DATA_TYPES = {
+    MLRS_DRAG_TO_REORDER: 'MLRS_DRAG_TO_REORDER',
+    MLRS_DRAG_TO_MOVE: 'MLRS_DRAG_TO_MOVE'
+};
+
+export const FAKE_VALUE = 'dnd_move_in_progress';
```

Comment:
I don't understand the purpose of this variable.
If you need to know when the user is dragging an element, you should use a boolean or a `useState`.

Reply:
It is actually set in the state (setMove()), the constant is used to prevent duplicating the value as I need to check the value in the ListView.